rpc/tls.go

```diff
@@ -113,6 +130,37 @@ func LoadInsecureTLSConfig() *TLSConfig {
 	}
 }
 
+// LoadClientTLSConfigFromDir creates a client TLSConfig by loading the root CA certs from the
+// specified directory. The directory must contain ca.crt.
+func LoadClientTLSConfigFromDir(certDir string) (*TLSConfig, error) {
+	caPEM, err := ioutil.ReadFile(path.Join(certDir, "ca.crt"))
+	if err != nil {
+		return nil, err
+	}
+	return LoadClientTLSConfig(caPEM)
+}
+
+// LoadClientTLSConfig creates a client TLSConfig from the supplied byte strings containing
+// the certificate of the cluster CA.
+func LoadClientTLSConfig(caPEM []byte) (*TLSConfig, error) {
+	certPool := x509.NewCertPool()
+
+	if ok := certPool.AppendCertsFromPEM(caPEM); !ok {
+		err := util.Error("failed to parse PEM data to pool")
+		return nil, err
+	}
+
+	return &TLSConfig{
```

Comment:
They do. The only flag suite flag left is PreferServerCipherSuites which is server-side only.